source/clear-linux/reference/bundles/openssh-server.rst

```diff
@@ -110,9 +110,35 @@ To enable root login via SSH, perform the following steps:
    .. code-block:: bash
 
       mkdir /etc/ssh
+      
+#. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
+   
+#. Set the configuration variable in /etc/ssh/sshd_config
 
-#. Set the configuration variable.
-
-   .. code-block:: bash
-
-      echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
+    .. code-block:: console
+    
+      PermitRootLogin yes
+      
+Enable X11-forwarding 
+*****************
+ To enable X11-forwarding via SSH, perform the following steps:
+ 
+ #. Create a *ssh* directory in :file:`/etc`, if it does not already exist.
+ 
+ .. code-block:: bash
+ 
+   mkdir /etc/ssh
+   
+ #. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
```

Comment:
I suggest following this up with a command the reader can issue so it's consistent with the format of the other steps. (i.e. $EDITOR /etc/ssh/sshd_config)